app/views/editor/cinematic/CinematicEditorComponent.vue

```diff
@@ -101,13 +103,17 @@ module.exports = Vue.extend({
           }
         })
       }
-
       const c = this.cinematic
       const data = $.extend(true, {}, c.attributes)
       const el = $(`<div></div>`);
+      const files = new CocoCollection(await api.files.getDirectory({path: 'cinematic'}), { model: Cinematic })
       const treema = this.treema = TreemaNode.make(el, {
         data: data,
         schema: Cinematic.schema,
+        // Automatically uploads the file to /file/cinematic/<fileName>
+        // You can view files at /admin/files
+        filePath: `cinematic`,
```

Comment:
Nitpick: I believe `standard` will turn this into a `'` based string.

Reply:
Great catch!